doc/api/http.md

```diff
@@ -1493,6 +1501,19 @@ added: v0.4.2
 Indicates that the underlying connection was closed.
 Just like `'end'`, this event occurs only once per response.
 
+#### message.aborted
+<!-- YAML
+added: REPLACEME
+-->
+
+* {boolean}
+
+<<<<<<< HEAD
```

Comment:
This should not have passed a `make test`. If so, we need to update `Makefile` to fix the conflict detection.

Reply:
That was just me being in a hurry. Apologies.